lib/components/Positioner/stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { select } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import React, { useCallback, useRef, useState } from 'react';
+import { Box } from '../Box';
+import { EAlignment } from './alignment';
+import { Positioner } from './Positioner';
+
+const alignmentPicker = () =>
+	select('Alignment', EAlignment, EAlignment.BOTTOM_LEFT);
+
+storiesOf('Utility|Positioner', module).add('default', () => {
```

Comment:
We should have a better story that chromatic can read, maybe put a delay on this story?